lib/livebook_web/live/session_live.ex

```diff
@@ -1688,6 +1700,27 @@ defmodule LivebookWeb.SessionLive do
     {:markdown, %{source: source}}
   end
 
+  defp cell_type_and_attrs_from_params(%{"type" => "form"}) do
+    source = """
+    form =
+      Kino.Control.form(
+        [
+          name: Kino.Input.text("Name"),
+        ],
+        submit: "Submit"
+      )
+
+    form
+    |> Kino.render()
+    |> Kino.Control.stream()
+    |> Kino.listen(fn event ->
+      IO.inspect(event)
+    end)
```

Comment:
To avoid the empty new line at the end:

```suggestion
    end)\
```